pymeasure/instruments/philips/PM6669.py

```diff
@@ -0,0 +1,102 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2021 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+from time import sleep
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set, strict_range
+
+class HardwareErrorException(Exception):
+    pass
+
+class PM6669(Instrument):
+    """ Represents the Philips PM 6669 instrument.
+    """
+
+
+    def __init__(self, resourceName, **kwargs):
+        super().__init__(
+            resourceName,
+            "Philips PM 6669",
+            **kwargs
+        )
+        self.write("EOI ON")
+
+    def wait_for_measurement(self, poll_interval = 1):
+        s = self.spoll()
+        while not (s &0b1):
+            sleep(poll_interval)
+            s = self.spoll()
+            if (s & 0x10000):
```

Comment:
One suggestion to make returns from `spoll()` a bit more readable is to implement an `IntFlag` type similar to what is done here: https://github.com/pymeasure/pymeasure/blob/f03315b3ae9149370ac4f6b00652e08ce8656add/pymeasure/instruments/srs/sr830.py#L34